documentation/frontend/featureFlags.md

```diff
@@ -0,0 +1,162 @@
+# Feature flags
+
+This feature flags implementation stores feature flags settings on user cookies. The intent is for these feature flags
+to be user configurable, so that 1. we can gate incomplete features to make it easier to coordinate development work,
+and 2. links can be sent to individuals with customized query parameters to customize the user's experience.
+
+For codebase/deployment level feature flagging that isn't meant for users to be able to configure, those continue to be
+implemented using `process.env`.
+
+## Feature flags in use
+
+As of February 2024, the following feature flags are set in the project:
+1. hideSearch - 
+  * Defaults to true
```

Comment:
Can we align as a team on defaulting to `true` or `false`? It'd also be good to avoid double-negatives in naming the flags. 

> Feature flags should be named so that the absence of a default value is interpreted as `false` – this way if someone forgets to define a feature flag, it doesn’t unintentionally enable the feature for everyone.
> 
> – https://github.com/navapbc/archive-massgov-pfml/blob/5b05735033d0c08d37271469e8978a2a04c43a54/docs/portal/feature-flags.md?plain=1#L11C1-L12C1

Reply:
I'd prefer vars like `show[Thing]` and `enable[Feature]`.

Reply:
nit: add a v0 suffix to a new FF name? Gives the flexibility of showing different versions or parts of search in the future